src/python/tests/test_orthology_benchmark.py

```diff
@@ -0,0 +1,224 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit testing of `orthology_benchmark.py` script.
+
+Typical usage example::
+
+    $ pytest test_orthology_benchmark.py
+
+"""
+
+from contextlib import nullcontext as does_not_raise
+from importlib.abc import Loader
+from importlib.machinery import ModuleSpec
+from importlib.util import module_from_spec, spec_from_file_location
+import os
+from pathlib import Path
+import sys
+from typing import ContextManager, Dict, List
+
+import sqlalchemy
+
+import pytest
+from pytest import raises
+
+
+script_path = Path(__file__).parents[3] / "scripts" / "pipeline" / "orthology_benchmark.py"
+script_name = script_path.stem
+script_spec = spec_from_file_location(script_name, script_path)
+
+if not isinstance(script_spec, ModuleSpec):
+    raise ImportError(f"ModuleSpec not created for module file '{script_path}'")
+if not isinstance(script_spec.loader, Loader):
+    raise ImportError(f"no loader found for module file '{script_path}'")
+
+orthology_benchmark_module = module_from_spec(script_spec)
+sys.modules[script_name] = orthology_benchmark_module
+script_spec.loader.exec_module(orthology_benchmark_module)
+
+# pylint: disable=import-error,wrong-import-position
+
+import orthology_benchmark  # type: ignore
+
+# pylint: enable=import-error,wrong-import-position
+
+from ensembl.compara.filesys import file_cmp
+
+
+@pytest.mark.parametrize(
+    "multi_dbs",
+    [
+        [{'src': 'core/gallus_gallus_core_99_6', 'name': 'gallus_gallus_core_99_6'},
+         {'src': 'core/homo_sapiens_core_99_38', 'name': 'homo_sapiens_core_99_38'}]
```

Comment:
I had to remind myself how it worked, but based on the code, what you have written is equivalent to:
```suggest
        [{'src': 'core/gallus_gallus_core_99_6'}, {'src': 'core/homo_sapiens_core_99_38'}]
```

Reply:
Nope. Travis fails so I reverted back to `[{'src': 'core/gallus_gallus_core_99_6', 'name': 'gallus_gallus_core_99_6'}, {'src': 'core/homo_sapiens_core_99_38', 'name': 'homo_sapiens_core_99_38'}]`. I think the major problem is that omitting `'name'` leads to test dbs with names `gallus_gallus, homo_sapiens`, i.e. without release and version info.

Reply:
OK, Travis fails again so I probably misinterpreted its messages. Let me try out your other suggestions and I will get back to this in the end.

Reply:
I decided to keep `'name'` for the sake of clarity and as a help for fellow colleagues and future generations who will use this script as a template for their tests.